hoomd/md/pytest/test_potential.py

```diff
@@ -452,6 +521,14 @@ def _update_snap(pair_potential, snap):
         snap.particles.diameter[1] = 0.5
 
 
+def _check_for_skip(sim, pair_potential):
```

Comment:
The way this method is used in the tests means that it could be used for any reason to skip a test. Right now, it only applies to triplet potentials, but as we add more types of potentials (like anisotropic), we may want to skip some of those tests in some cases as well.

Reply:
Right but I would prefer to have a few functions that skip for different reasons than one function that skips them for multiple as the former I more clear to future developers.

Reply:
I think having one helper function looks cleaner in the calling code rather than calling each of the many helper functions. I don't think its really a big deal at this point, if we need to change it later we can.

Reply:
Yes, the name should be clearer for what this skips. I cannot imagine how one function could possibly handle all cases. Different tests will have different configurations that need to be skipped. The name should be explicit: `skip_if_triplet_gpu_mpi`.